client/models/messageeventmodel.cpp

```diff
@@ -59,6 +60,9 @@ QHash<int, QByteArray> MessageEventModel::roleNames() const
         roles.insert(EventResolvedTypeRole, "eventResolvedType");
         roles.insert(RefRole, "refId");
         roles.insert(ReactionsRole, "reactions");
+        roles.insert(NudeRichBodyRole, "nudeRichBody");
```

Comment:
"Nude" is not often applied to inanimate objects, AFAIK; "bare" is.
```suggestion
        roles.insert(BareRichBodyRole, "bareRichBody");
```